src/style.css

```diff
@@ -1 +1,90 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+/*test shows 6-2. please help to realize my mistakes*/
```

Comment:
I don't see the reason to test fails, try to fix everything I've suggested

It looks as expected in the demo, so if after your changes test still fails - it's not a big problem in this case